modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service for the most recent graph of update possibilities.
+This graph is based on the channel to which the cluster is subscribed.
+If the graph contains paths to OCP versions that are newer than the cluster's current version, the CVO saves these versions into either the `availableUpdates` or `conditionalUpdates` field of its ClusterVersion resource.
+
+The CVO periodically checks the conditional updates for update risks.
+These risks are conveyed through the data served by the OpenShift Update Service, which contains information for each version about known issues that may affect a cluster updated to that version.
+Most risks are limited to clusters with specific characteristics, such as clusters with a certain size or clusters that are deployed in a particular cloud platform.
+
+The CVO continuously evaluates its cluster characteristics against the conditional risk information for each conditional update, and if it finds that the cluster matches the criteria, it stores this information in the `conditionalUpdates` field of its ClusterVersion resource.
+If the CVO finds that the cluster doesn't match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its ClusterVersion resource.
+The user interface (web console and CLI) then presents this information to the administrator.
+Each "supported but not recommended" update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
+
+You can inspect all available updates with the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --include-not-recommended
```

Comment:
nit: I agree with @petr-muller 's [descriptive vs. instructive distinction][1].  But in dipping into instruction here, can we either add some wiggle room like "One way to inspect ...", or just skip the instructive detail?  As it stands, it feels a bit weird to me to cover the `oc` interface without also covering the web-console interface.

[1]: https://github.com/openshift/openshift-docs/pull/60091#discussion_r1204006628

Reply:
I'll loosen up the wording to indicate that this is just one way to do it.